### ISSUE TITLE
Circle Pad doesn't work

### ISSUE THREAD

**Issue by Carbuino** [elhobbs/prboom3ds#14]
So, I can only move using the D-Pad and Circle Pad does nothing.

**Comment by elhobbs** [comment]
There is a key bindings menu.

**Comment by FateForWindows** [comment]
Yes. As elhobbs said, by default the movement keys are bound to the dpad. You will have to manually change them to use the circle pad.

**Comment by JennaScvl** [comment]
> Yes. As elhobbs said, by default the movement keys are bound to the dpad. You will have to manually change them to use the circle pad.

Ok? but how?
elhobbs says "key bindings menu". I have tried to use the menu, it does not respond to key bindings. Manually change them? Ok... but how? I know there's a config file... I can easily open it in a text editor... But if you say to just edit the config file, you're not providing the information about WHAT exactly to edit it to say.

Don't say "read the docs". I've done that. I've read the docs. I know all about how to change the resolution on Windows and Linux, neither of which are operating systems present on the 3DS or even valid for the context of this specific branch...

The key bindings menu responds to dpad. It responds to button presses. It even responds to the c-stick. It does not respond to circle pad. Since the circle pad is not bound to anything in the config file and the documentation does not mention what the bind names for the circle pad actually are, I can't help but feel the both of you are giving advice without actually bothering to consider the issue, never mind understand it.

If I sound hostile, I'm not, I'm just frustrated with the snippy, off handed advice that's hardly any better than "have you tried turning it off and back on again?"

**Comment by FateForWindows** [comment]
Are you using the 3dsx or CIA? The latest CIA works perfectly for me.

**Comment by JennaScvl** [comment]
I am... REALLY sorry... this is going to sound really dumb on my part, but it actually worked when I turned it off and back on again. I've narrowed it down to being caused by either BootNTR or Luma's input redirection. I figured this out because cpad wasn't working period until I rebooted and the only thing I did besides prboom was bootntr and luma... I'm thinking it was also caused by an experimental program I've been making which talks to BootNTR and Luma from the computer...

So it's all my fault and I apologize. I deleted my original comment.
